Source/WebKit/UIProcess/API/glib/WebKitNavigationPolicyDecision.cpp

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
Oops, you added an extra blank line at the top.

```suggestion
```

Reply:
My keyboard is doing weird things with the enter key :-/